src/safe_apps/migrations/0008_tag.py

```diff
@@ -0,0 +1,32 @@
+# Generated by Django 4.0.3 on 2022-04-06 13:14
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("safe_apps", "0007_safeapp_access_policy"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="Tag",
+            fields=[
+                (
+                    "id",
+                    models.BigAutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                ("name", models.CharField(max_length=255)),
```

Comment:
Oh, so it _is_ a model with id and name. The joke's on me :D

Reply:
This is so that a `Tag` can be tied to multiple `SafeApps`. If we need to update a tag for all the apps then we just need to edit in one place.